api.go

```diff
@@ -15,15 +15,16 @@ import (
 )
 
 const (
-	// This is the current suggested max size of the data in a raft log entry.
-	// This is based on current architecture, default timing, etc. Clients can
+	// SuggestedMaxDataSize of the data in a raft log entry, in bytes.
+	//
+	// The value is based on current architecture, default timing, etc. Clients can
 	// ignore this value if they want as there is no actual hard checking
 	// within the library. As the library is enhanced this value may change
 	// over time to reflect current suggested maximums.
 	//
-	// Increasing beyond this risks RPC IO taking too long and preventing
-	// timely heartbeat signals which are sent in serial in current transports,
-	// potentially causing leadership instability.
+	// Applying logs entries with data greater than this size risks RPC IO taking
+	// too long and preventing timely heartbeat signals which are sent in serial
```

Comment:
```suggestion
	// too long and preventing timely heartbeat signals.  These signals are sent in serial
```

(bit of a run-on sentence)